examples/app_server_with_auto_scaler/app.py

```diff
@@ -0,0 +1,86 @@
+import warnings  # FIXME: remove before merge
+from typing import Any, List
+
+import torch
+import torchvision
+from pydantic import BaseModel
+
+import lightning as L
+
+warnings.filterwarnings("ignore", "Arguments other than a weight enum", UserWarning)  # FIXME: remove before merge
+warnings.filterwarnings("ignore", "The parameter 'pretrained' is deprecated", UserWarning)  # FIXME: remove before merge
+
+
+class RequestModel(BaseModel):
+    image: str
+
+
+class BatchRequestModel(BaseModel):
+    inputs: List[RequestModel]
+
+
+class BatchResponse(BaseModel):
+    outputs: List[Any]
+
+
+class PyTorchServer(L.app.components.PythonServer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            port=L.app.utilities.network.find_free_network_port(),
+            input_type=BatchRequestModel,
+            output_type=BatchResponse,
+            cloud_compute=L.CloudCompute("gpu"),
+        )
+
+    def setup(self):
+        self._model = torchvision.models.resnet18(pretrained=True)
+        self._device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+        self._model.to(self._device)
```

Comment:
```suggestion
        self._device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
        self._model = torchvision.models.resnet18(pretrained=True).to(self._device)
```

Reply:
IMHO, I want the example to be as simple as possible, so I wouldn't put device as an optional argument.

Reply:
:+1: